src/org/omegat/gui/dialogs/VersionCheckDialog.java

```diff
@@ -61,8 +61,11 @@ protected void done() {
                         new VersionCheckDialog(VersionChecker.getInstance().getRemoteVersion()).show(parent);
                     }
                 } catch (Exception e) {
-                    Log.log(e);
-                    JOptionPane.showMessageDialog(parent, e.getLocalizedMessage(), OStrings.getString("ERROR_TITLE"),
+                    Log.logWarningRB("VERSION_CHECK_FAILED");
+                    Log.log(e.toString());
+                    JOptionPane.showMessageDialog(parent,
+                            OStrings.getString("VERSION_CHECK_FAILED"),
```

Comment:
If you want to leave out the details of the error here, I guess we need a separate string. However since this dialog is only shown when the user explicitly requests a version check, not explaining the reason seems inconsiderate to me. But I don't feel that strongly about it either way.